ext/test/fail_fs/fail_fs.c

```diff
@@ -239,11 +239,13 @@ fail_file_read(
     } else
         read_ops = ++fail_fs->read_ops;
 
+    if (fail_fs->fail_enabled && fail_fs->allow_reads != 0 &&
+      read_ops % fail_fs->allow_reads == 0) {
+        ret = fail_fs_simulate_fail(fail_fh, session, read_ops, "read");
+        goto err;
```

Comment:
```suggestion
        WT_ERR(fail_fs_simulate_fail(fail_fh, session, read_ops, "read"););
```

Reply:
The fail_fs.c file is outside of WiredTiger source code. The WT_ERR() macro is able to used here. I was following the file's other examples of err labels and they use goto err: explicitly. What do you think should be the correct style for this file? @etienneptl

Reply:
I don't think we should use `WT_ERR` if this is not internal code, good call.